<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFunctionsDialog</class>
 <widget class="QDialog" name="AddFunctionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="filterOptions">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Regular Expression</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Function Name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Start and End Time</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="filterOptionsLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter Syntax</string>
   </property>
  </widget>
  <widget class="QLabel" name="filterStringLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter String</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="filterString">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="filterStartLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="filterEndLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>End Time</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="filterTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>491</width>
     <height>221</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Select</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Start Time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>End Time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QCheckBox" name="selectAllFunctions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="filterStart">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>10000000000000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="filterEnd">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>10000000000000000000.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddFunctionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddFunctionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
